packages/app/obojobo-express/routes/api/locks.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express')
+const router = express.Router()
+const EditLock = oboRequire('models/edit_lock')
+const { userHasPermissionToDraft } = require('obojobo-repository/server/services/permissions')
+const { checkValidationRules, requireDraftId, requireCanViewEditor } = oboRequire(
+	'express_validators'
+)
+
+// CHECK A LOCK
+// mounted as /api/locks/:draftId
+router
+	.route('/:draftId')
+	.get([requireDraftId, requireCanViewEditor, checkValidationRules])
+	.get(async (req, res) => {
```

Comment:
Should we call `EditLock.deleteExpiredLocks()` here? Otherwise it seems old locks never get cleared out unless someone tries to create one.